rails-install-ubuntu.sh

```diff
@@ -2,50 +2,69 @@
 
 set -e
 
+echo "Adding PPA for up-to-date Node.js runtime. Give your password when asked."
+sudo add-apt-repository ppa:chris-lea/node.js
```

Comment:
Good call on Node; I forgot about the js runtime. 

Why are you using a third-party repo (chris-lea ppa) instead of:

```
curl -sL https://deb.nodesource.com/setup | sudo bash -
sudo apt-get install -y nodejs
```
?